tensorflow/core/lib/io/inputbuffer.h

```diff
@@ -75,6 +75,13 @@ class InputBuffer {
   // read will trigger a File::Read().
   Status Seek(int64 position);
 
+  // Hint to the total read size within the file before next Seek.
+  //
+  // If there are enough data present in the buffer, do nothing.
```

Comment:
Remove the second paragraph. The documentation should not include implementation details.